server/planning/io/ingest_rule_handler.py

```diff
@@ -174,7 +176,7 @@ def process_autopost(self, ingest_item: Dict[str, Any]):
         item_id = ingest_item.get(config.ID_FIELD)
         update_post_item(
             {
-                "pubstatus": ingest_item.get("pubstatus") or POST_STATE.USABLE,
+                "pubstatus": ingest_item.get("ingest_pubstatus") or POST_STATE.USABLE,
```

Comment:
I would probably add a second fallback here, so it would be `ingest_pubstatus` or `pubstatus` or `POST_STATE.USABLE`